bulletin/index.html

```diff
@@ -177,9 +177,55 @@ <h3 class="buarticle-scholarships--title">MACILE Scholarships Program </h3>
 				<img src="/images/bulletin/scholarships/andreina_troncoso.jpg">
 				<p>Andreina De Jesús T., 9th grade, Colegio Padre Zegri, Nigua.</p>
 			</article>
-			
 		</div>	
-	</div>
+	</div> 
+	<section class="feature-article">
+		<div class="feature-article--wrapper">
+			<h2 class="feature-article--section-heading">Featured Article</h2>
+			<div class="article">
+				<div class="article-banner">
+					<h1 class="article-banner--brand" itemprop="brand">MACILE</h1> 
+					<!-- <h3 class="article-banner--slogan">Advancing Engineering and Technology Education</h3> -->
+				</div>
+				<div class="feature-article__content">
+					<h2 class="article-title"> Engineering Robust Learning Environments Through Continuous Improvemnt</h2>
+					<p class="article-abstract"> 
+							MACILE, founded in 2007, is a research-based education program created to advance engineering, technology, math, and science education in less advantaged communities. One core objective is to design solutions to optimally increase access to robust language-infused STEM learning environments. Development follows three phases: exploration, design refinement, and expansion. We review the trajectory and accomplishments from 2007 and 2008. Clearly, emphasis on continuous improvement is producing astoundingly encouraging results.
+					</p>
+				</div>	
+			</div>
+		</div>
+	</section>
+	<section class="feature-comment">
+		<div class="feature-article--wrapper">
+			<h2 class="feature-comment-heading">From Our Community</h2>
+			<div class="comment">
+				<div class="feature-comment-item">
+					<blockquote>This program helped me fall in love with mathematics and language. It gave me new dreams and hopes. I have no words to express my gratitude to MACILE for bringing us what I like best, science, and for instilling in me a passion for learning... - MACILE Student</blockquote>
+				</div>
+				<div class="feature-comment-item">
+					<blockquote>This summer, I learned to read better, how to express my thoughts, and how to analyze problems… -MACILE Student
+				    </blockquote>
+				</div>
+				<div class="feature-comment-item">
+					<blockquote>I learned many examples of good behavior, capitalization…
+					</blockquote>	
+				</div>
+				<div class="feature-comment-item">
+					<blockquote>I learned a lot about bridges, solar energy, mathematics, and other things. I also learned that I should not speak with a loud voice and not to disturb others. I also learned that working on a team is extremely important to achieve my goals.
+					</blockquote>
+				</div>
+				<div class="feature-comment-item">
+						<blockquote> I learned that I should ask questions when in doubt. If I remain silent, I will never learn what I need to learn. I also learned some mathematics and Spanish. I did not know how to write an essay. I used to make a textual copy of the material.
+
+						</blockquote>
+				</div>
+			</div>
+		</div>
+	</section>
+	<section class="feature-acknowledge">
+
+	</section>
```

Comment:
What is this section? Why is it empty?

Reply:
No sure why is showing here. I removed it in my local repo of the branch.  
Could it be because the backup.html file is untracked?

Reply:
Why do you think it would have to do with an untracked file? Should an untrack file affect anything?

Reply:
Because it does not show in index.html in my local branch. It is in the file backup.html, which is not tracked.